lib/elastic_apm/subscriber.rb

```diff
@@ -35,6 +37,9 @@ def start(name, id, payload)
 
       normalized = @normalizers.normalize(transaction, name, payload)
 
+      _caller = caller
```

Comment:
`caller` is a method, so we save the result in a variable to avoid calling it again.